docs/integrations/logging.md

```diff
@@ -0,0 +1,6 @@
+# Logging
```

Comment:
@RealOrangeOne anything more you'd want to cover here?

Reply:
I think we should add a little bit on how to log, specifically how to put small bits of data into the message, and using `extra` to include more context

Reply:
53c1ed55